worker/src/RTC/RtpStreamSend.cpp

```diff
@@ -483,15 +483,15 @@ namespace RTC
 		}
 
 		// Only clone once and only if necessary.
-		if (!*clonedPacket)
+		if (!clonedPacket.get())
 		{
-			*clonedPacket = packet->Clone();
+			clonedPacket = packet->Clone();
```

Comment:
The old version updates the value in the reference. Will this change `clonedPacket` for the caller though? I'm not a C++ expert, but it looks like `clonedPacket` is only updated within this function, which means it will be cloned multiple instead of once.

Reply:
This is now enhanced within the last changes. The `clonedPacket` is now just reseted within a `RtpPacket` pointer (only for the first consumer needing it), but the instance is not modified by any means.